### DISCUSSION TITLE
There may be an error in 03_Action.ipynb

### DISCUSSION THREAD

**Discussion by epetrishchev** [rte-france/Grid2Op#264] [General]
Hello. In the process of getting acquainted with 03_Action.ipynb, I drew attention to this piece of code:

```
gen_id = 1 # on which generator to apply this action
ratio_curtailment = 0.15 # this is a ratio, between 0.0 and 1.0
# # if not in range [0, 1.0] an "ambiguous action" will be released when calling "env.step"
curtail_act = action_space({"curtail": [(gen_id, ratio_curtailment)]})
```
and

```
curtail_act2 = action_space()
curtail_act2.curtail = [(gen_id, amount)]
```

In theory, these are different ways to do the same thing. But the variable `amount` is used in the second method, which leads to a change in the power of the generator by 314% of the maximum power.

I apologize if I described the error in the wrong place, can you tell me where to write about the inaccuracies found next time. Thanks.

**Comment by BDonnot** [comment]
Hello,

This is the perfect place for such feedback.

I will create an issue from this example. 

Thanks for reporting it

Benjamin
